examples/line_plots/multi_line_plot.py

```diff
@@ -1,42 +1,48 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from emcpy.plots.plots import LinePlot
-from emcpy.plots.create_plots import CreatePlot
+from emcpy.plots.create_plots import CreatePlot, CreateFigure
 
-x1 = [0, 401, 1039, 2774, 2408]
-x2 = [500, 250, 710, 1515, 1212]
-x3 = [400, 150, 910, 1215, 850]
-y1 = [0, 2.5, 5, 7.5, 12.5]
-y2 = [1, 5, 6, 8, 10]
-y3 = [1, 4, 5.5, 9, 10.5]
 
-# Create line plot object
+def _getLineData():
+    # generate test data for line plots
+
+    x1 = [0, 401, 1039, 2774, 2408]
+    x2 = [500, 250, 710, 1515, 1212]
+    x3 = [400, 150, 910, 1215, 850]
+    y1 = [0, 2.5, 5, 7.5, 12.5]
+    y2 = [1, 5, 6, 8, 10]
+    y3 = [1, 4, 5.5, 9, 10.5]
+
+    return x1, y1, x2, y2, x3, y3
+
+
+# create line plot with two sets of axes
+# sharing a common y axis
+
+x1, y1, x2, y2, x3, y3 = _getLineData()
 lp1 = LinePlot(x1, y1)
 lp1.label = 'line 1'
 
-# Create line 2 plot object
 lp2 = LinePlot(x2, y2)
 lp2.color = 'tab:green'
 lp2.label = 'line 2'
+lp2.use_shared_ay()
 
-# Create line 3 plot object
 lp3 = LinePlot(x3, y3)
 lp3.color = 'tab:red'
 lp3.label = 'line 3'
+lp3.use_shared_ay()
 
-# Add line plot objects to list
-plt_list = [lp1, lp2, lp3]
-
-# Create Plot and draw data
 myplt = CreatePlot()
+plt_list = [lp1, lp2, lp3]
```

Comment:
Does the order in which this list is created matter?

Reply:
No, the order of the pot objects should not have any impact on the plot.  Though now that you ask I wonder if an attempt to plot on the shared_x or shared_y axes before anything is plotted on the default ax axes would result in a problem.  I'll test that tomorrow.  If there's a problem I'll open a new issue for it.

Reply:
I ran some tests and can confirm the order of the object list does not matter.